### PR TITLE
refcator, chore(egen) : refactored code according to template guidelines , performed linter test

### PR DESCRIPTION
**Pull request by Jayakrishna2801**
This PR includes 
 1. changed REGION variable to LOCATION
 2. added colab enterprise link
 3. performed linter test
 4. removed contradiction words 

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb [hunk 1]
```diff
@@ -142,156 +135,51 @@
...
-        "REGION = \"us-central1\"  # @param {type: \"string\"}"
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "dr--iN2kAylZ"
-      },
-      "source": [
-        "### Authenticate your Google Cloud account\n",
```
Comment (katiemn) [inline notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb:231]: Keep the authentication section, format according to the template

File: notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb [hunk 2]
```diff
@@ -354,9 +230,9 @@
         "id": "intro_gar"
       },
       "source": [
-        "## Introduction to Google Artifact Registry\n",
+        "## Introduction to Google Artifact registry\n",
```
Comment (katiemn) [inline notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb:233]: capitalize Registry

File: notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb [hunk 3]
```diff
@@ -367,9 +243,9 @@
         "id": "gar_enable_api"
       },
       "source": [
-        "### Enable Artifact Registry API\n",
+        "### Enable Artifact registry API\n",
```
Comment (katiemn) [inline notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb:246]: capitalize Registry

File: notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb [hunk 4]
```diff
@@ -367,9 +243,9 @@
...
         "\n",
-        "First, you must enable the Artifact Registry API service for your project.\n",
+        "First,enable the Artifact Registry API service for your project.\n",
```
Comment (katiemn) [inline notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb:248]: add a space between 'First,' and 'enable'

File: notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb [hunk 5]
```diff
@@ -539,7 +415,7 @@
       "source": [
         "### Deleting your private Docker repostory\n",
         "\n",
-        "Finally, once your private repository becomes obsolete, use the command `gcloud artifacts repositories delete` to delete it `Google Artifact Registry`."
+        "Finally, once your private repository becomes obsolete, use the command gcloud artifacts repositories delete to delete it Google Artifact Registry."
```
Comment (katiemn) [inline notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb:418]: Modify to: "Finally, once your private repository becomes obsolete, use the `gcloud artifacts repositories delete` command to delete the Google Artifact Registry."

File: notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb [hunk 6]
```diff
@@ -356,7 +257,7 @@
       "source": [
         "## Introduction to Google Artifact Registry\n",
         "\n",
-        "The `Google Artifact Registry` is a service for storing and managing artifacts in private repositories, including container images, Helm charts, and language packages. It is the recommended container image registry for Google Cloud.\n",
+        "The Google Artifact Registry is a service for storing and managing artifacts in private repositories, including container images, Helm charts, and language packages. It is the recommended container image registry for Google Cloud.\n",
```
Comment (kittyabs) [inline notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb:260]: contract "It is" -> "It's"

File: notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb [hunk 7]
```diff
@@ -471,7 +372,7 @@
         "\n",
         "- Tagging the Docker image with a repository name configures the docker push command to push the image to a specific location, e.g., us-central1-docker.pkg.dev.\n",
         "\n",
-        "- `:my-tag` is a tag you're adding to the Docker image. If a tag is not specified, it defaults to `:latest`."
+        "- :my-tag is a tag you're adding to the Docker image. it defaults to :latest."
```
Comment (kittyabs) [inline notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb:375]: upper case "it" -> "It defaults to..."

File: notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb [hunk 8]
```diff
@@ -539,7 +440,7 @@
       "source": [
         "### Deleting your private Docker repostory\n",
         "\n",
-        "Finally, once your private repository becomes obsolete, use the command `gcloud artifacts repositories delete` to delete it `Google Artifact Registry`."
+        "Finally, once your private repository becomes obsolete, use the `gcloud artifacts repositories delete` command to delete it Google Artifact Registry."
```
Comment (kittyabs) [inline notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb:443]: seems "remove....from the" is missing:

- "Finally, once your private repository becomes obsolete, use the `gcloud artifacts repositories delete` command to remove the repository from the Google Artifact Registry."

**Review by kittyabs — COMMENTED** [review]
numerous edits

File: notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb [hunk 9]
```diff
@@ -142,156 +135,76 @@
...
-      },
-      "outputs": [],
-      "source": [
-        "PROJECT_ID = \"[your-project-id]\"  # @param {type:\"string\"}\n",
+        "    import IPython\n",
         "\n",
-        "# Set the project id\n",
-        "! gcloud config set project {PROJECT_ID}"
+        "    app = IPython.Application.instance()\n",
+        "    app.kernel.do_shutdown(True)"
```
Comment (katiemn) [inline notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb:160]: Please add the restart warning from the template after this code cell

**Review by katiemn — APPROVED** [review]

**Pull request merged**